apps/src/code-studio/assets/DefaultSpritesEditor.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  getDefaultList,
+  updateDefaultList
+} from '@cdo/apps/assetManagement/animationLibraryApi';
+import DefaultSpriteRow from '@cdo/apps/code-studio/assets/DefaultSpriteRow';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+
+export default class DefaultSpritesEditor extends React.Component {
+  state = {
+    isLoading: true,
+    defaultList: {}, // Dictionary with name as key and sprite object as value
+    pendingChanges: 0,
```

Comment:
naming nit: pendingChanges makes me think this is tracking the changes themselves, but it's actually tracking the count of pending changes -- maybe rename to something like `pendingChangesCount`?